docs/UserGuide.md

```diff
@@ -376,23 +382,23 @@ This is the list of features that help you manage the records of doctor:
 - [Editing doctor](#editing-a-doctor--edit_d)
 - [Finding doctor](#locating-doctors-by-name--find_d)
 - [Deleting doctor](#deleting-a-doctor--delete_d)
-- [Adding Specialisation](#adding-a-specialisation--add_tag_d)
-- [Deleting Specialisation](#deleting-a-specialisation--delete_tag_d)
-- [Editing Remark](#editing-a-remark--remark_d)
+- [Adding Specialisation](#adding-a-specialisation-to-a-doctor--add_tag_d)
+- [Deleting Specialisation](#deleting-a-specialisation-from-a-doctor--delete_tag_d)
+- [Editing Remark](#editing-the-remark-of-a-doctor--remark_d)
```

Comment:
should standardize the naming of the commands with appointments. e.g. in appointment its `Adding a medical condition to a patient` but here is `Adding Specialisation`